src/pages/token-metadata/print.md

```diff
@@ -10,7 +10,7 @@ Every NFT has the potential to be printed as multiple editions when its **Master
 
 The owner of a printable NFT can print as many editions as they want from it as long as its maximum supply has not been reached.
 
-Every Non-Fungible asset — i.e. `NonFungible` and `ProgrammableNonFungible` token standards — can be a printable NFT when created. This is done by configuring the **Max Supply** attribute of the asset's Master Edition account. This attribute is optional and can have one of the following values:
+Every Non-Fungible Nft — i.e. `NonFungible` and `ProgrammableNonFungible` token standards — can be a printable NFT when created. This is done by configuring the **Max Supply** attribute of the Nft's Master Edition account. This attribute is optional and can have one of the following values:
```

Comment:
I think the term Non-Fungible Nft is redundant so maybe this is another one to leave as asset?